### ISSUE TITLE
Permissions request screen not showing for calendar and camera access.

### ISSUE THREAD

**Issue by pneves001** [BaseflowIT/flutter-permission-handler#153]
## 🐛 Bug Report

My app is not  generating a calendar permission request.  

### Expected behavior

Permission handler should ask for permissions for ALL of the services I'm requesting permissions. 

### Reproduction steps

I'm using the following code to request permisisons. 

```
permissions = await PermissionHandler().requestPermissions(
                            [
                            PermissionGroup.calendar,
                            PermissionGroup.contacts,
                            PermissionGroup.storage,
                            PermissionGroup.camera, 
                            PermissionGroup.photos
                            ]);

print(permissions.toString());                   

if ((permissions[PermissionGroup.contacts] != PermissionStatus.granted) ||
     (permissions[PermissionGroup.calendar] != PermissionStatus.granted) ||
     (permissions[PermissionGroup.storage] != PermissionStatus.granted) ||
     (permissions[PermissionGroup.camera]  != PermissionStatus.granted) || 
     (permissions[PermissionGroup.photos]  != PermissionStatus.granted))
{
    bool isOpened = await PermissionHandler().openAppSettings();  
}
```

The app I'm developing currently needs access contacts, calendar, camera, filestorage, photos, and camera.

### Configuration

```
version: 1.0.0+1

environment:
  sdk: ">=2.1.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  permission_handler: ^3.2.2
  font_awesome_flutter: ^8.5.0
  firebase_core: any
  firebase_auth: 0.14.0+5
  flutter_facebook_login: ^2.0.1
  google_sign_in: ^4.0.7
  image_picker: 0.6.1+4
  contacts_service: ^0.3.0
  intl: ^0.15.7
  flutter_calendar_carousel: ^1.3.17
  http: ^0.12.0+2
  device_calendar: 1.0.0+2
  web_socket_channel: ^1.0.15
  network_image_to_byte: ^0.0.1
  # The following adds the Cupertino Icons font to your application.
  # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^0.1.2

dev_dependencies:
  flutter_test:
    sdk: flutter
```  

Al

**Version:** 1.x

**Platform:** 

- [ X] :robot: Android

**Comment by mvanbeusekom** [comment]
@pneves001, thank you for this report. Unfortunately I cannot reproduce this issue. I have added the following code to the example App to simulate a test and a pop-up is shown for each of the permissions I requested:

```
RaisedButton(
  child: Text('Test'),
  onPressed: () {
    PermissionHandler().requestPermissions([
      PermissionGroup.calendar,
      PermissionGroup.contacts,
      PermissionGroup.storage,
      PermissionGroup.camera, 
      PermissionGroup.photos,
      ]).then((result) {
        print(result.toString());
      });
  },
),
```

Are you sure you have added all the required permissions to the `AndroidManifest.xml` (note that there is a `debug`, `main` and `profile` version which is selected depending on how you run the App, e.g. in debug, release or profile mode)?

**Comment by pneves001** [comment]
<uses-permission android:name="android.permission.INTERNET"/>
    <uses-permission android:name="android.permission.READ_CONTACTS" />  
    <uses-permission android:name="android.permission.WRITE_CONTACTS" /> 
    <uses-permission android:name="android.permission.READ_CALENDAR" />
    <uses-permission android:name="android.permission.WRITE_CALENDAR" /> 
    <uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.STORAGE" />

**Comment by pneves001** [comment]
Acutally thank you. I found intents in the main section which fixed it.

**Comment by mvanbeusekom** [comment]
Nice that you solved it. Could you give some more info on the intents? There are other users with similar problems which might also be helped with this solution.

**Comment by pneves001** [comment]
So it was as you said there was a set of intents set in the AndroidManifest.xml file inside the main folder. Not the debug or profile folder. That set of intents didn't have the intents I set in my profile AndroidManifest.xml.  So when I moved my intents into the main folder. It all started working.  That might be something to add in the documentation. That this might not work if you place the intents in the wrong AndroidManifest.xml file.
